pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -21,6 +21,8 @@ const (
 
 	RouterUsername   = "system:" + RouterUnqualifiedUsername
 	RegistryUsername = "system:" + RegistryUnqualifiedUsername
+
+	BuildControllerUsername = "system:serviceaccount:openshift-infra:build-controller"
```

Comment:
this shouldn't be a constant, the namespace is configurable and there's a helper function to build the name